WatchParty/client/src/components/Header/Header.js

```diff
@@ -1,43 +1,96 @@
 import React from 'react';
+import { Route, Switch, withRouter} from "react-router-dom"
 import { Nav, Navbar, NavDropdown } from 'react-bootstrap'
 import { Form, FormControl, Button } from 'react-bootstrap'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
+import SearchResults from "./SearchResults"
```

Comment:
Shouldn't the `SearchResults` component be instantiated in the `App` component? I know sure why it's being lumped together with `Header`